### PR TITLE
[14.0][PORT] stock_quant_package_dimension  PR 777 from 13.0

### PR DESCRIPTION
**Pull request by hparfr**
Add estimated_pack_weight_kg to stock.quant.package

### CODE REVIEW

**Comment by rousseldenis** [comment]
@hparfr Could you try to fix duplicate labels too ?

image

**Comment by hparfr** [comment]
@rousseldenis  duplicate label fixed

@Kev-roche can you please review

**Review by Kev-Roche — APPROVED** [review]
LGTM, I allow myself to fix a typo

File: stock_quant_package_dimension/models/stock_quant_package.py [hunk 1]
```diff
@@ -6,18 +6,24 @@
 class StockQuantPackage(models.Model):
     _inherit = "stock.quant.package"
 
-    pack_weight = fields.Float("Weight")
-    pack_length = fields.Integer("Length", help="length")
-    width = fields.Integer("Width", help="width")
-    height = fields.Integer("Height", help="height")
+    pack_weight = fields.Float("Pack Weight")
```
Comment (rousseldenis) [inline stock_quant_package_dimension/models/stock_quant_package.py]: You can remove string attribute.

```suggestion
    pack_weight = fields.Float()
```

**Review by rousseldenis — APPROVED** [review]
Minor comment

**Pull request merged**